worlds/ExportWorld.py

```diff
@@ -0,0 +1,100 @@
+# TODO: handle data
+# TODO: handle clients
+# TODO: handle yaml templates
+import argparse
+import json
+import os.path
+import platform
+import sys
+import zipfile
+from pathlib import Path
+
+if __name__ == "__main__":
+    sys.path.remove(os.path.dirname(__file__))
+    ap_root = os.path.normpath(os.path.join(os.path.dirname(__file__), os.pardir))
+    os.chdir(ap_root)
+    sys.path.append(ap_root)
+
+from worlds.AutoWorld import AutoWorldRegister, World
+
+METADATA_KEYS = [
+    "game",
+    # license
+    # description
+    # maintainers
+]
```

Comment:
something that should be in metadata: What file types can this apworld handle?
so that we don't need to load every world to find out whether we have a handler for `.apz5` files

We can just load the one world that knows how to handle the one file we're trying to open.